.github/workflows/test.yml

```diff
@@ -204,6 +204,18 @@ jobs:
     name: Test Longer-running Motion Plans if affected
     uses: viamrobotics/rdk/.github/workflows/motion-tests.yml@main
 
+  jsonschema:
+    name: Test meta.json jsonschema
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-go@v5
+    - run: |
+        go run ./cli/viam/ module create --local-only --name schematest --public-namespace ns
+        cat meta.json | jq '.models = [{"api": "x:y:z", "model": "x:y:z"}]' > meta2.json
+        pip install check-jsonschema
+        check-jsonschema --schemafile ./cli/module.schema.json meta2.json
```

Comment:
[opt] How do we feel about breaking this out into two tests:

1) One tests in the cli dir that uses the [already-imported jsonschema library](https://github.com/viamrobotics/rdk/blob/682a0439fea77341089340b5e7eef2f78f3e2e65/go.mod#L52) to ensure that the schema is valid
2) Another test (possibly in this repo, possibly in declarative-infra) that verifies that the dl.viam.com shortlink is functioning as expected

This would mean that we don't need the `--local-only` flag & we could keep the cli tests clustered.

Reply:
(I don't feel too strongly about this though and this LGTM as-is if you think this is the right way to go)

Reply:
Admittedly, that jsonschema library is a bit overpowered for this use case, I suspect but could still do the job of verifying correctness

Reply:
oh there's a golang one; didn't know

the golang package looks like it's for generating jsonschema from golang struct -- can it check as well?

also we're not testing the dl.viam.dev shortlink anywhere

Reply:
> the golang package looks like it's for generating jsonschema from golang struct -- can it check as well?

The package is mostly for generating, however, it also contains a struct type `jsonschema.Schema` and I suspect a  unmarshall + marshall + ensureEqual could catch the bugs that are likely to occur from changes to this file. 

(we could then also add some tests to ensure that the tags in the human-written schema matches the generated jsonschema from the struct... but up to you)

However, I don't know what `check-jsonschema` actually checks so it might catch other bugs.

> also we're not testing the dl.viam.dev shortlink anywhere

Ah you're right. I misread this and had assumed it was checking the link, not the file.

Reply:
talked offline -- decided it was fine as-is